run/tests/http/async/main.go

```diff
@@ -18,6 +18,7 @@ func main() {
 	go func() {
 		handler := http.HandlerFunc(callback)
 		http.Handle("/callback", handler)
+		//nolint:gosec
```

Comment:
Is this necessary, if we configure it in the yml?

Reply:
normally not, but it is not picking it up from the config. and I don't know why. since this is not an important test, I just want to go on here.